library/core/src/fmt/mod.rs

```diff
@@ -267,7 +267,7 @@ extern "C" {
 #[allow(missing_debug_implementations)]
 #[unstable(feature = "fmt_internals", reason = "internal to format_args!", issue = "none")]
 #[doc(hidden)]
-pub struct ArgumentV1<'a> {
+pub struct Argument<'a> {
```

Comment:
Is this still used in any macro? Can we make it private now that it's a lang item?